options/locale/locale_en-US.ini

```
@@ -1723,7 +1723,7 @@ pulls.no_merge_desc = This pull request cannot be merged because all repository
 pulls.no_merge_helper = Enable merge options in the repository settings or merge the pull request manually.
 pulls.no_merge_wip = This pull request cannot be merged because it is marked as being a work in progress.
 pulls.no_merge_not_ready = This pull request is not ready to be merged, check review status and status checks.
-pulls.no_merge_access = You are not authorized to merge this pull request.
+pulls.no_merge_access = You are not authorized to merge This pull request.
```

Did you use a regex?
```suggestion
pulls.no_merge_access = You are not authorized to merge this pull request.
```

---

Presumably unsuccessfully, I tried to go over it once again after I noticed that I messed some things up but I did not correct everything back, it seems.